pages/common/sui.md

```diff
@@ -0,0 +1,24 @@
+# sui
+
+> Interact with the Sui network.
+> More information: <https://docs.sui.io/references/cli/cheatsheet>.
+
+- Execute a sui subcommand:
+
+`sui {{subcommand}}`
+
+- Build tools for a smart contract:
+
+`sui move {{subcommand}}`
+
+- Client to interact with the Sui network:
```

Comment:
The descriptions of examples should be written in the imperative mood (see [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood)).

Reply:
Missed this one, thanks a lot! I updated it. ✅